### PR TITLE
Add Cardfight!! Vanguard G: Stride to Victory!!

### PR DESCRIPTION
**Pull request by CHShadow**
Adds Cardfight!! Vanguard G: Stride to Victory!!.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/cardfight-vanguard-g-stride-to-victory/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Cardfight!! Vanguard G: Stride to Victory!!"
+description = "Cardfight!! Vanguard G: Stride to Victory!! is a collectable card game developed and published by FuRyu based off the hit trading card game Cardfight! Vanguard by Bushiroad. This is the 3rd Nintendo 3ds game in the series and includes cards up until the the 4th G Booster Set plus Promo Cards. The game takes you through the story of G as a new cardfighter."
```
Comment (Leo626) [inline games/cardfight-vanguard-g-stride-to-victory/game.dat]: * In the first sentence, the second `Cardfight!` should have two exclamation marks, i.e. `Cardfight!!`.
* In the second sentence, `3ds` should be stylized as `3DS`.

Comment (CHShadow) [reply games/cardfight-vanguard-g-stride-to-victory/game.dat]: new to github, how do i change the box art picture?

**Comment by CodingKoopa** [comment]
Please address the review comments, or this will be closed.

**Comment by CodingKoopa** [comment]
Closing due to inactivity.

**Pull request closed without merging**